recipes/flask-table/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "flask-table" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 320e5756cd7252e902e03b6cd1087f2c7ebc31364341b482f41f30074d10a770
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - flask
```

Comment:
It is unfortunate that there is no `flask` version dependency, but it matches what is [upstream's setup.py](https://github.com/plumdog/flask_table/blob/09175a2e040ff41f25d9bf50a44ae6c789430d00/setup.py#L6)